boards/banglejs-2/include/periph_conf.h

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright (C) 2019 Inria
+ *               2022 Nicole Faerber, Siegen, Germany
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_banglejs2
+ * @{
+ *
+ * @file
+ * @brief       Peripheral configuration for the Bangle.js 2
+ *
+ * @author      Nicole Faerber <nicole.faerber@digitaluhr-manufactur.de>
+ *
+ */
+
+#ifndef PERIPH_CONF_H
+#define PERIPH_CONF_H
+
+#include "periph_cpu.h"
+#include "cfg_clock_32_1.h"
+#include "cfg_rtt_default.h"
+#include "cfg_timer_default.h"
+#include "board.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name    I2C configuration
+ * @{
+ */
+static const i2c_conf_t i2c_config[] = {
+    {
+        // CST816S touschscreen
```

Comment:
```suggestion
        /* CST816S touschscreen */
```

Yeah, I know. C knows about the since at least C99 and I truely have no idea why our coding convention still doesn't allow them. But I guess it is just less effort to just use plain ANSI C comments than start that discussion.